src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -43,6 +43,8 @@ iOS
 
 This API retrieves the previously fetched propositions, for the provided decision scopes, from the in-memory extension propositions cache. The completion callback is invoked with the decision propositions corresponding to the given decision scopes. If a certain decision scope has not already been fetched prior to this API call, it will not be contained in the returned propositions.
 
+In case of [**result simlation**](./review-simulate.md#simulate-different-results) this method will return simulated results.
```

Comment:
Can we reword this to `When connected to Assurance for previewing content, this method....`